priv/repo/migrations/20161020144622_create_stripe_account.exs

```diff
@@ -0,0 +1,28 @@
+defmodule CodeCorps.Repo.Migrations.CreateStripeAccount do
+  use Ecto.Migration
+
+  def change do
+    create table(:stripe_accounts, primary_key: false) do
```

Comment:
We want the id to be a string, because this is a stripe object, which uses strings as ids. As far as I can tell, this is the only way to achieve that. Otherwise, by default, the `:id` will be of `:id` type, not `:string`